src/OpenSage.Tools.BigEditor/Util/ImGuiUtility.cs

```diff
@@ -7,7 +7,7 @@ internal static class ImGuiUtility
     {
         public static unsafe bool InputText(string label, byte[] textBuffer, out string result)
         {
-            var temp = ImGui.InputText(label, textBuffer, (uint) textBuffer.Length, ImGuiInputTextFlags.None, data => 0);
+            var temp = ImGui.InputText(label, textBuffer, (uint)textBuffer.Length, ImGuiInputTextFlags.None, data => 0);
```

Comment:
Reverse this whitespace change.

```suggestion
            var temp = ImGui.InputText(label, textBuffer, (uint) textBuffer.Length, ImGuiInputTextFlags.None, data => 0);
```